WORKSPACE

```diff
@@ -0,0 +1,31 @@
+# Copyright 2017-present The Material Foundation Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+git_repository(
+    name = "build_bazel_rules_apple",
+    remote = "https://github.com/bazelbuild/rules_apple.git",
+    commit = "7ea0557",
+)
+
+git_repository(
+    name = "bazel_ios_warnings",
+    remote = "https://github.com/material-foundation/bazel_ios_warnings.git",
+    tag = "v1.0.1",
+)
+
+git_repository(
+    name = "material_internationalization_ios",
+    remote = "https://github.com/jverkoey/material-internationalization-ios.git",
```

Comment:
Should this URL point to a Material organizational account?  It looks like it's currently pointing to your fork.

Reply:
Done.